src/Diagnostics/atmos_mass_energy_loss.jl

```diff
@@ -0,0 +1,120 @@
+# AtmosMassEnergyLoss
+#
+# Dump mass and energy loss
+
+using ..Atmos
+using ..BalanceLaws
+using ..Mesh.Topologies
+using ..Mesh.Grids
+using ..MPIStateArrays
+
+"""
+    setup_atmos_mass_energy_loss(
+        ::ClimateMachineConfigType,
+        interval::String,
+        out_prefix::String,
+        writer = NetCDFWriter(),
+        interpol = nothing,
+    )
+
+Create and return a `DiagnosticsGroup` containing the
+"AtmosMassEnergyLoss" diagnostics for Atmos LES and GCM
+configurations. All the diagnostics in the group will run at the
+specified `interval`, be interpolated to the specified boundaries
+and resolution, and written to files prefixed by `out_prefix`
+using `writer`.
+"""
+function setup_atmos_mass_energy_loss(
+    ::ClimateMachineConfigType,
+    interval::String,
+    out_prefix::String,
+    writer = NetCDFWriter(),
+    interpol = nothing,
+)
+    @assert isnothing(interpol)
+
+    return DiagnosticsGroup(
+        "AtmosMassEnergyLoss",
+        Diagnostics.atmos_mass_energy_loss_init,
+        Diagnostics.atmos_mass_energy_loss_fini,
+        Diagnostics.atmos_mass_energy_loss_collect,
+        interval,
+        out_prefix,
+        writer,
+        interpol,
+    )
+end
+
+# store initial mass and energy
+Base.@kwdef mutable struct AtmosMassEnergyLoss
+    Σρ₀::Float64 = 0.0
```

Comment:
Should this be explicitly Float64 or should we be consistent with the type used across the code?